src/utils/Validator.ts

```diff
@@ -0,0 +1,15 @@
+export class Validator {
+  // tslint:disable-next-line: no-empty
+  public constructor() {}
+
+  /**
+   * Validate query or path parameter provided in the request
+   * @param parameter Query or path parameter provided
+   * @returns boolean Result of logic determining validity
+   */
+  public parameterIsValid(parameter: string): boolean {
+    return parameter
+      ? parameter.trim().length !== 0 && parameter !== 'undefined' && parameter !== 'null'
```

Comment:
This is allowing an empty string to go through ("") and return a 200 response (activities/open?testerStaffId="")

Reply:
/activities/open?testerStaffId="" is not an empty string, it's a string of 2 special characters: ""

As far as I can tell, null or undefined should never be 'passed' through as a path parameter, that would simply be an empty path paramater. However I did create a number of tests to validate that it would work, on line 73 and 89 of the openVisitFunction unit tests.